app/services/tagged_content.rb

```diff
@@ -27,7 +27,7 @@ def search_response
       count: RummagerSearch::PAGE_SIZE_TO_GET_EVERYTHING,
```

Comment:
Sorry to be a pain, given that I suggested this approach, but if we have for example, 2 content ids and the first one brings back >= `PAGE_SIZE_TO_GET_EVERYTHING` documents, will rummager ignore the second content id, or does it merge the two results, order them and then send us back the first x number of them?

I know we currently have `PAGE_SIZE_TO_GET_EVERYTHING` set to 1000 so pretty big, but just want to be sure we aren't going to get surprising results here.